src/main/webapp/style.css

```diff
@@ -1,17 +1,17 @@
-#help-near-me-content {
+.all-content {
```

Comment:
should be named something more specific like maintainer organizations content or something